biothings/hub/databuild/builder.py

```diff
@@ -864,6 +864,8 @@ async def merge_source(self, src_name, batch_size=100000, ids=None, job_manager=
         meta = src_master.find_one({"_id": src_name}) or {}
         merger = meta.get("merger", "upsert")
         self.logger.info("Documents from source '%s' will be merged using %s", src_name, merger)
+        merger_keywords = meta.get("merger_keywords")
```

Comment:
just to follow the typical naming convention, you might want to call it `merger_kwargs` instead